docs/DEVELOPER_GUIDE.md

```diff
@@ -52,12 +52,17 @@ You must install these tools:
 
 ### Install Knative on a Kubernetes cluster
 
-1. [Set up a kubernetes cluster and install Knative Serving](https://knative.dev/docs/install/)
+KFServing currently requires `Knative Serving` for auto-scaling, canary rollout, `Istio` for traffic routing and ingress.
 
-- **Note**: KFServing currently only requires `Knative Serving` for auto-scaling, canary rollout,
- `Istio` for traffic routing and ingress. You can follow instructions on
- [Custom Install](https://knative.dev/docs/install/knative-custom-install) to install `Istio` and `Knative Serving`,
- observability plug-ins are good to have for monitoring. If you already have `Istio` (e.g. from a kubeflow install) then simply skip the `Istio` steps.
+* You can follow the instructions on [Set up a kubernetes cluster and install Knative Serving](https://knative.dev/docs/install/) or
+[Custom Install](https://knative.dev/docs/install/knative-custom-install) to install `Istio` and `Knative Serving`. Observability plug-ins are good to have for monitoring.
+
+* If you already have `Istio` (e.g. from a Kubeflow install) then simply skip the `Istio` steps. For Kubeflow install, you can install `Knative Serving` via
+the following commands after downloading repository [kubeflow/manifests](https://github.com/kubeflow/manifests).
```

Comment:
May be specify that version of knative in manifests is 0.8

Reply:
Done

Reply:
Any reason why native 0.9 does not work with kubeflow 0.6.2?

Reply:
The `autoscaler` pod of Knative 0.9 is always in CrashBackOff loop after being installed on Kubeflow v0.6.2.

I don't know the reason as no logs is shown for the pod. but my guess is there is some Istio setting of KF doesn't work with knative 0.9 out of the box. That's why I start look into manifests repo to see if there some knative config that works with current KF.

Reply:
@zhenghuiwang you should be able to get the logs with `--previous` flag, I am curious about the error. Also which version of Istio kubeflow is using ?